GPU/Common/ReplacedTexture.cpp

```diff
@@ -54,55 +54,68 @@ class ReplacedTextureTask : public Task {
 	LimitedWaitable *waitable_;
 };
 
+// This can only return true if ACTIVE or NOT_FOUND.
 bool ReplacedTexture::IsReady(double budget) {
 	_assert_(vfs_ != nullptr);
-	lastUsed_ = time_now_d();
-	if (threadWaitable_ && !threadWaitable_->WaitFor(budget)) {
-		return false;
-	}
 
-	// Loaded already, or not yet on a thread?
-	if (initDone_ && levelData_ && !levelData_->data.empty()) {
-		// TODO: lock?
-		levelData_->lastUsed = lastUsed_;
```

Comment:
It looks like `ReplacedLevelsCache::lastUsed` is no longer used, should probably be removed.

-[Unknown]

Reply:
Possibly, though, it might make sense to use it still. I'll re-evaluate later.

Reply:
Actually I think I somehow got confused here and overlooked it, though it was the texture lastUsed.